### ISSUE TITLE
Move StorefrontRender subscriber to the bottom of the listeners

### ISSUE THREAD

**Issue by dloop-mb** [webmatch/WbmTagManagerEcomm#69]
### PHP Version

7.4

### Shopware Version

6.4.5.1

### Expected behaviour

The WbmTagManagerEcomm subscriber to the [StorefrontRenderEvent](https://github.com/webmatch/WbmTagManagerEcomm/blob/94035c63664b9d7e09c355208cb549f86b8e16c7/src/Subscriber/StorefrontRenderSubscriber.php#L43) should be compatible with any other subscriber to this event.

### Actual behaviour

If another plugin is installed/activated that subscribes to the same event but runs later and tries to add a global variable to the twig environment via the [addGlobal](https://github.com/twigphp/Twig/blob/31629a69f7a074e44e3f180b3940a579f56ed3fb/src/Environment.php#L745) method then it throws a [LogicException](https://github.com/twigphp/Twig/blob/31629a69f7a074e44e3f180b3940a579f56ed3fb/src/Environment.php#L748).
The problem is that the [renderDataLayer](https://github.com/webmatch/WbmTagManagerEcomm/blob/94035c63664b9d7e09c355208cb549f86b8e16c7/src/Subscriber/StorefrontRenderSubscriber.php#L67) method closes the twig environment to accept further global variables (and probably other extensions).

### How to reproduce in a demoshop

Install the WbmTagManagerEcomm plugin and another plugin that subscribes to the StorefrontRenderEvent but runs later and tries to add a global twig variable.

**Comment by dloop-mb** [comment]
My solution suggestion is:
Add a [priority](https://symfony.com/doc/5.3/event_dispatcher.html#creating-an-event-subscriber) of -1 to the subscriber, so that it will run after all subscribers that use the default priority of 0 (or higher).

This change to the [subscriber](https://github.com/webmatch/WbmTagManagerEcomm/blob/94035c63664b9d7e09c355208cb549f86b8e16c7/src/Subscriber/StorefrontRenderSubscriber.php#L40) removed the exception in our environment:
```
    public static function getSubscribedEvents(): array
    {
        return [
            StorefrontRenderEvent::class => ['onRender', -1],
        ];
    }
```

**Comment by wbm-sbasler** [comment]
Will be fixed in v2.2.1
see #72
